pkg/pod/entrypoint.go

```diff
@@ -141,7 +141,7 @@ func orderContainers(entrypointImage string, steps []corev1.Container) (corev1.C
 func UpdateReady(kubeclient kubernetes.Interface, pod corev1.Pod) error {
 	newPod, err := kubeclient.CoreV1().Pods(pod.Namespace).Get(pod.Name, metav1.GetOptions{})
 	if err != nil {
-		return fmt.Errorf("Error getting Pod %q when updating ready annotation: %w", pod.Name, err)
+		return fmt.Errorf("Error getting Pod %q when updating ready annotation: %v", pod.Name, err)
```

Comment:
Were these changes intended to be part of this PR?

Reply:
Well I'm not sure if it has something to do with go 1.13, but the complier was complaining about those.

Reply:
I'll revert that change though.